lib/index.js

```diff
@@ -20,9 +20,11 @@ var AdRoll = module.exports = integration('AdRoll')
   .global('__adroll_loaded')
   .global('adroll_adv_id')
   .global('adroll_custom_data')
+  .global('adroll_email')
   .global('adroll_pix_id')
   .option('advId', '')
   .option('pixId', '')
+  .option('v1', false)
```

Comment:
It's basically the same thing as this option—it's a flag that should not be exposed to the user. Basically suggestion is, rather than a `bool` it should be an `int`. Otherwise we're creating a singleton option; if we ever need to make another one of these changes, we have to create an entirely new option and migrate everyone instead of just grabbing `version=3`

Maybe we should get in the habit of prefixing internal options with an underscore, ala `_version`, to make it super clear.